scp/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ file(GLOB_RECURSE HEADERS "*.hpp")
 list(FILTER SOURCES EXCLUDE REGEX ".*/tests/.*")
 list(FILTER HEADERS EXCLUDE REGEX ".*/tests/.*")
 
+set(CMAKE_C_FLAGS_RELEASE "-DSCP_DEBUG")
+set(CMAKE_CXX_FLAGS_RELEASE "-DSCP_DEBUG")
```

Comment:
is there any advantage of this over 'add_definitions(-DSCP_DEBUG)'?
why debug definition uses this CMAKE_FLAGS and other definitions use add_definition?
why can't all 3 definitions use common style?

Reply:
okay